wiki/Tournaments/Official_support/en.md

```diff
@@ -167,6 +167,9 @@ Profile badge images must adhere to the following standards:
 - It must clearly display the logo, motif, or name of your tournament.
 - It must be cleanly designed and of reasonable visual quality.
 - It must not include any sort of sponsorship or promote anything besides the tournament.
+- It must not include any characters or logos which were not specifically made for the tournament and adheres to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).
```

Comment:
```suggestion
- It must not include any media which was not specifically made for the tournament and must adhere to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).
```
thoughts on this wording

Reply:
"media" isn't really a correct substitution. if you want one word it'd be something like "artworks" or "designs"

(also note the plural)

Reply:
perhaps assets then, since id imagine things like “hype videos” as you call it would be covered too

Reply:
Please keep in mind that these changes are being specifically made under the "Badges" header so we're not trying to cover every design possible here.

Reply:
Ah my mistake, then assets alone seems like good wording I think to cover many bases